tests/dummy/config/environment.js

```diff
@@ -16,7 +19,9 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
-    }
+    },
+
+    ACCEPTANCE_TESTS: fs.readdirSync(path.join(__dirname, '..', 'app', 'acceptance', 'tests'))
```

Comment:
This config only applies to the addon's dummy testing app, and shouldn't ever be executed when it's included in any other application.

...unless I just have a fundamental misunderstanding of how the addon infrastructure works? It's totally possible ;)